engines/grim/imuse/imuse_track.cpp

```diff
@@ -133,21 +133,21 @@ bool Imuse::startSound(const char *soundName, int volGroupId, int hookId, int vo
 	track->curRegion = -1;
 	track->trackId = l;
 
-	int bits = 0, freq = 0, channels = 0;
 
 	Common::strcpy_s(track->soundName, soundName);
 	track->soundDesc = _sound->openSound(soundName, volGroupId);
 
 	if (!track->soundDesc)
 		return false;
 
-	bits = _sound->getBits(track->soundDesc);
-	channels = _sound->getChannels(track->soundDesc);
-	freq = _sound->getFreq(track->soundDesc);
+	const int bits = _sound->getBits(track->soundDesc);
+	const int channels = _sound->getChannels(track->soundDesc);
+	const int freq = _sound->getFreq(track->soundDesc);
 
 	assert(bits == 8 || bits == 12 || bits == 16);
 	assert(channels == 1 || channels == 2);
 	assert(0 < freq && freq <= 65535);
+	(void)bits;
```

Comment:
@aquadran Should it be used? The others are used.

Reply:
you can remove bits usage

Reply:
Do you mean not call getBits, or accept this suppression?

Reply:
you can accept suppression